### ISSUE TITLE
Issues with installation and first start up

### ISSUE THREAD

**Issue by dotcarls** [Levelmoney/generator-megatome#1]
I don't have a great grasp on how all the different components in this project work together yet, so forgive me if I'm just not doing something right. 

That said, during the initial generation when it begins to run `npm install` I receive the following error:
```bash
npm ERR! peerinvalid The package react@0.14.3 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-async@2.1.0 wants react@>=0.12.0 <0.14.0
```
I changed the version in `package.json` so that react 0.13.3 is used. Then I attempted to spin up the dev server, `npm run up` at which point this happens:
```bash
MacDaddy:docker tim$ npm run up

> xxxx@0.0.0 up /Users/tim/Dev/docker
> BUILD_DEV=1 webpack-dev-server --watch --progress --colors --port 8000 --inline --content-base dist/ 

 70% 2/2 build moduleshttp://localhost:8000/
webpack result is served from http://localhost:8000/
content is served from /Users/tim/Dev/docker/dist
404s will fallback to /index.html
Hash: deb7fce0f7aa81c77553  
Version: webpack 1.12.6
Time: 2705ms
     Asset       Size  Chunks             Chunk Names
    app.js     270 kB       0  [emitted]  app
   libs.js    1.98 MB       1  [emitted]  libs
index.html  810 bytes          [emitted]  
chunk    {0} app.js (app) 218 kB [rendered]
    [0] multi app 76 bytes {0} [built] [1 error]
     + 61 hidden modules
chunk    {1} libs.js (libs) 1.8 MB [rendered]
    [0] multi libs 148 bytes {1} [built]
     + 391 hidden modules

ERROR in ./src/bootstrap.js
Module build failed: SyntaxError: /Users/tim/Dev/docker/src/bootstrap.js: Unexpected token (21:17)
  19 |   Router.HistoryLocation,
  20 |   function (Handler) {
> 21 |     React.render(<Handler/>, mount);
     |                  ^
  22 | });
  23 | 
    at Parser.pp.raise (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/location.js:22:13)
    at Parser.pp.unexpected (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/util.js:91:8)
    at Parser.pp.parseExprAtom (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:510:12)
    at Parser.pp.parseExprSubscripts (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:265:19)
    at Parser.pp.parseMaybeUnary (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:245:19)
    at Parser.pp.parseExprOps (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:176:19)
    at Parser.pp.parseMaybeConditional (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:158:19)
    at Parser.pp.parseMaybeAssign (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:121:19)
    at Parser.pp.parseExprListItem (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:988:16)
    at Parser.pp.parseCallExpressionArguments (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:341:20)
    at Parser.pp.parseSubscripts (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:304:29)
    at Parser.pp.parseExprSubscripts (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:275:15)
    at Parser.pp.parseMaybeUnary (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:245:19)
    at Parser.pp.parseExprOps (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:176:19)
    at Parser.pp.parseMaybeConditional (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:158:19)
    at Parser.pp.parseMaybeAssign (/Users/tim/Dev/docker/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:121:19)
 @ multi app
webpack: bundle is now VALID.
```
So I'm guessing the reference error is from a breaking API change caused by the conflicting versions? Or not.. and I'm just quite confused.

**Comment by puppybits** [comment]
The newer versions of React and React Router had breaking changes. I've just pushed an updated generator that includes the latest versions of everything.

Let me know if you have any problems with it.

**Comment by dotcarls** [comment]
Awesome, thank you for the quick response. Just a heads up, had to install with `npm install -g git+https://github.com/Levelmoney/generator-megatome.git' as NPM has not been updated yet. Not sure if that's something that propagates on its own or if you need to do something. Thanks again!

**Comment by puppybits** [comment]
Thanks! I forgot to publish. I just published it to npm (v1.0.0).
